big-dipper/imports/api/notifications/server/methods.js

```diff
@@ -1,27 +1,27 @@
-import { Meteor } from 'meteor/meteor'
-import { WebApp } from 'meteor/webapp'
-import { Notifications } from '../notifications.js'
-import { FCMToken } from '../../fcmtoken/fcmtoken.js'
-import { isNumber, isString } from 'lodash'
-import { sanitizeUrl } from '@braintree/sanitize-url'
-import { HTTP } from 'meteor/http'
-import { admin } from '../../admin.js'
-import connectRoute from 'connect-route'
+import { Meteor } from "meteor/meteor";
+import { WebApp } from "meteor/webapp";
+import { Notifications } from "../notifications.js";
+import { FCMToken } from "../../fcmtoken/fcmtoken.js";
+import { isNumber, isString } from "lodash";
+import { sanitizeUrl } from "@braintree/sanitize-url";
+import { HTTP } from "meteor/http";
+import { admin } from "../../admin.js";
+import connectRoute from "connect-route";
 
-const StatusOk = 200
-const StatusInvalidInput = 400
-const Success = 'Success'
-const BadRequest = 'Bad Request'
-const InvalidID = 'Invalid Notification ID'
-const AppCheckFailed = 'App Check Failed'
+const StatusOk = 200;
+const StatusInvalidInput = 400;
+const Success = "Success";
+const BadRequest = "Bad Request";
+const InvalidID = "Invalid Notification ID";
+const AppCheckFailed = "App Check Failed";
```

Comment:
I thought we discussed there was no need to use appcheck in big dipper? How are we using appcheck?

Reply:
Yes so we need the FCM token from our app so for that we have applied app-check verification before we set FCM. Plus it enhances a bit in terms of security.

Reply:
Is this resolved?